client/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "smee-cli",
+  "version": "1.0.0",
+  "description": "Client to proxy webhooks to local host",
+  "main": "index.js",
+  "bin": {
+    "smee": "./bin/smee.js"
+  },
+  "scripts": {
+    "test": "jest --coverage && standard"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/probot/webhooks.git"
```

Comment:
Should we changes this to point to `probot/smee` (and then change the repo itself of course)?

Reply:
🔜